src/main/scala/nl.knaw.dans.easy.archivebag/EasyArchiveBag.scala

```diff
@@ -64,6 +64,9 @@ object EasyArchiveBag extends Bagit5FacadeComponent with DebugEnhancedLogging {
         }
         zippedBag.delete()
         location
+      case HttpStatus.SC_BAD_REQUEST =>
+        logger.error(s"${ ps.storageDepositService } returned:[ ${ response.getStatusLine } ]. ZippedBag=$zippedBag reason = ${ response.getStatusLine.getReasonPhrase }")
```

Comment:
this case looks very similar to the one below in `case _ =>`. Only an extra `reason = ${ response.getStatusLine.getReasonPhrase }` was added. However, I wonder whether you now introduce a duplicate `reasonPhrase`. Here's the reason why I suspect so (without having tested this):

* you first got `returned:[ ${ response.getStatusLine }].` in this `String`, so implicitly `response.getStatusLine.toString` is called.
* the `StatusLine` returned by `response.getStatusLine` is an interface with only one implementation, called `BasicStatusLine`.
* The `toString` of `BasicStatusLine` is implemented as follows:
    ```java
    @Override
    public String toString() {
        return BasicLineFormatter.INSTANCE.formatStatusLine(null, this).toString(); // here 'this' is the StatusLine itself
    }
    ```
* `formatStatusLine` calls `doFormatStatusLine`:
    ```java
    @Override
    public CharArrayBuffer formatStatusLine(final CharArrayBuffer buffer,
                                            final StatusLine statline) {
        Args.notNull(statline, "Status line");
        final CharArrayBuffer result = initBuffer(buffer);
        doFormatStatusLine(result, statline);
    
        return result;
    }
    ```
* `doFormatStatusLine` appends the `reasonPhrase` to the `CharArrayBuffer`:
    ```java
    final String reason = statline.getReasonPhrase();
    if (reason != null) {
        buffer.append(reason);
    }
    ```
* **Hence, if a `reason` is present (a.k.a. NOT `null`), then is should already be printed with the full status line. There is therefore no reason to add it yet another time to the `logger.error` text.**